pkg/detectors/uri/uri.go

```diff
@@ -19,85 +24,114 @@ type Scanner struct {
 }
 
 // Ensure the Scanner satisfies the interface at compile time.
-var _ detectors.Detector = (*Scanner)(nil)
-var _ detectors.CustomFalsePositiveChecker = (*Scanner)(nil)
+var _ interface {
+	detectors.Detector
+	detectors.CustomFalsePositiveChecker
```

Comment:
Is there any reason why we ignore all false positives for this detector with custom logic?

Reply:
I suspect there were too many results being filtered by the wordlist, so someone decided to do this workaround instead of removing the problematic entries.

See https://github.com/trufflesecurity/trufflehog/issues/3246